libs/environment/src/utils/compile-environment.ts

```diff
@@ -82,6 +82,8 @@ const getBundledEnvironmentValue = (key: EnvKey) => {
       return (
         process.env['MAINTENANCE_PAGE'] || process.env['NX_MAINTENANCE_PAGE']
       );
+    case 'DELEGATIONS_PAGINATION':
+      return process.env['NX_DELEGATIONS_PAGINATION'];
```

Comment:
chore (blocking): the environemnet provider is for things shared across apps. I would just use this in the env.ts file rather than here